app/assets/javascripts/foreman_google/json_loader.js

```diff
@@ -0,0 +1,15 @@
+const jsonLoader = input => {
+  const file = input.files[0];
+  const reader = new FileReader();
+  reader.onload = () => {
+    const text = reader.result;
+    const outputTextField = document.getElementById('gce_json');
+    outputTextField.value = text;
+  };
+  reader.readAsText(file);
+};
+
+window.gce = { jsonLoader };
+window.tfm = Object.assign(window.tfm || {}, {
+  gce: { jsonLoader },
+});
```

Comment:
the `tfm` object is generated by webpack
this file is served by the rails assets pipeline, so no need to override the tfm object.